### ISSUE TITLE
Payment provider name inconsistently translated

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#1923]
### Expected behavior
All payment provider names should either be translated or not translated (probably not translated?)

### Actual Behavior
Some are and some are not. 
![reaction](https://cloud.githubusercontent.com/assets/72819/24530874/742d9592-15e7-11e7-8e0f-2fb84a5d826d.png)

### Steps to Reproduce the Behavior
1. Log in as an admin
1. Go to Payment in the dashboard
1. Set language to be something besides English
1. Observe that some payment providers are translated, some are not.

**Comment by aaronjudd** [comment]
PayPal etc probably are brands.. not translatable word? not sure though..

**Comment by zenweasel** [comment]
Right, but Stripe should probably not be translated, as well as Braintree.

**Comment by zenweasel** [comment]
"Example Payment Method" could be translated, since it's not brand

**Comment by aaronjudd** [comment]
True. Is there a translation in i18n?

**Comment by zenweasel** [comment]
```
[{
  "language": "Arabic",
  "i18n": "ar",
  "ns": "reaction-stripe",
  "translation": {
    "reaction-payments": {
      "admin": {
        "shortcut": {
          "stripeLabel": "شريط"
        },
        "dashboard": {
          "stripeLabel": "شريط",
          "stripeDescription": "المدفوعات شريط"
        },
        "paymentSettings": {
          "stripeLabel": "شريط",
          "stripeSettingsLabel": "شريط",
          "stripeSettingsDescription": "لم يكن لديك معرف العميل الشريط API؟",
          "stripeSettingsGetItHere": "أحضره هنا"
        }
      }
    }
  }
}]
```

**Comment by zenweasel** [comment]
Trying to change these in LingoHub (I had already modified a bunch of these in code but some I can't really fix). But when I go into LingoHub the word shows as "locked" with the status "ordered". Have a support request in with LIngoHub

![editor___lingohub](https://cloud.githubusercontent.com/assets/72819/24784874/18229ae4-1b88-11e7-8943-8b94208fe98b.png)

**Comment by aaronjudd** [comment]
I actually think you need to leave a message for their support, I think that we paid for some translations that were never fulfilled...

**Comment by zenweasel** [comment]
I have been working in Lingo hub to set up "Terms" which you can specify global translations but I keep getting distracted or need to ask their support a question so I get delayed a day to wait for their response. So progress is happening, just super slowly.

**Comment by zenweasel** [comment]
I didn't finish this in time for the 1.1.1 release so moving to 1.2

**Comment by aaronjudd** [comment]
@spencern this is the issue where the payment providers were being corrected (discussed re: #2167)

@zenweasel issue with LingoHub is now resolved, and these translations can also be fixed there.

**Comment by zenweasel** [comment]
So I added terms for items below and marked them "untranslatable". That seems to work well. I just don't know how to trigger a PR from that change. I have a question in with support. But that seems like it will take care of it.

Let me know if you think of more:

Authorize.net
Braintree
PayPal/PayPal Express/ Paypal Pro
Shippo
Stripe
Avalara
TaxCloud

**Comment by zenweasel** [comment]
Closed via #2218
